ClothesApp/Application/Dtos/Categories/CategoryInputDto.cs

```diff
@@ -0,0 +1,8 @@
+namespace Application.Dtos.Category;
+
+public class CategoryInputDto
+{
+    public long ParentCategoryId { get; set; }
```

Comment:
What will you pass if it's top level category?

Reply:
I used to pass 0. Now null